config/elasticsearch.yml

```diff
@@ -29,28 +28,25 @@ discovery.zen.publish_timeout: ${DISCOVERY_ZEN_PUBLISH_TIMEOUT}
 discovery.zen.ping.unicast.hosts: ${DISCOVERY_ZEN_UNICAST_HOST}
 discovery.zen.minimum_master_nodes: ${DISCOVERY_ZEN_MINIMUM_MASTER_NODES}
 
-searchguard.ssl.transport.enabled: ${ENABLE_TRANSPORT_SSL}
-searchguard.ssl.transport.keystore_filepath: certs/keystore.jks
-searchguard.ssl.transport.keystore_alias: cert
-searchguard.ssl.transport.resolve_hostname: false
-
-searchguard.ssl.transport.truststore_filepath: certs/truststore.jks
-searchguard.ssl.transport.truststore_alias: ca
-
-thread_pool.bulk.queue_size: ${THREAD_POOL_BULK_QUEUE_SIZE}
+thread_pool.write.queue_size: ${THREAD_POOL_QUEUE_SIZE}
 indices.memory.index_buffer_size: ${INDEX_BUFFER_SIZE}
 
 xpack.security.enabled: ${XPACK_SECURITY_ENABLE}
+xpack.security.audit.enabled: ${XPACK_SECURITY_AUDIT_ENABLE}
+
 xpack.security.transport.ssl.enabled: ${XPACK_SECURITY_TRANSPORT_SSL_ENABLE}
+xpack.security.transport.ssl.verification_mode: ${XPACK_SSL_VERIFY_MODE}
+xpack.security.transport.ssl.key: ${XPACK_SSL_KEY_PATH}
+xpack.security.transport.ssl.certificate: ${XPACK_SSL_CERT_PATH}
+xpack.security.transport.ssl.certificate_authorities: [ "${XPACK_SSL_CA_CERT_PATH}" ]
+xpack.security.transport.ssl.supported_protocols: "${XPACK_SECURITY_TRANSPORT_SSL_PROTOCOLS}"
+
 xpack.security.http.ssl.enabled: ${XPACK_SECURITY_HTTP_SSL_ENABLE}
-xpack.security.transport.ssl.verification_mode: certificate
-xpack.security.audit.enabled: ${XPACK_SECURITY_AUDIT_ENABLE}
-xpack.security.audit.index.events.exclude: ${XPACK_SECURITY_AUDIT_INDEX_EVENTS_EXCLUDE}
```

Comment:
Has the support for excluding audit events been dropped in this version completely, both for index and logfile outputs?